relativedeltafield/__init__.py

```diff
@@ -134,18 +197,21 @@ def get_db_prep_value(self, value, connection, prepared=False):
 	# We can't simply replace or remove PsycoPg2's parser, because
 	# that would mess with any existing Django DurationFields, since
 	# Django assumes PsycoPg2 returns pre-parsed datetime.timedeltas.
+	# 	def select_format(self, compiler, sql, params):
+	# 		fmt = 'to_char(%s, \'PYYYY"Y"MM"M"DD"DT"HH24"H"MI"M"SS.US"S"\')' % sql
+	# 		return fmt, params
```

Comment:
Please don't comment out code that's unused. Just drop it instead. It can always be retrieved from version control history.

Reply:
Sorry it was a left orver